### PR TITLE
add DZ17VSA to equipment list

### PR DESCRIPTION
**Pull request by wirecuttr**
added heat pumpt to readme

### CODE REVIEW

File: custom_components/daikinone/daikinone.py [hunk 1]
```diff
@@ -289,7 +289,7 @@ def __map_equipment(self, payload: DaikinDeviceDataResponse) -> dict[str, Daikin
         if payload.data["ctAHUnitType"] < 255:
             model = payload.data["ctAHModelNoCharacter1_15"].strip()
             serial = payload.data["ctAHSerialNoCharacter1_15"].strip()
-            eid = f"{model}-{serial}"
```
Comment (zlangbert) [inline custom_components/daikinone/daikinone.py:292]: @wirecuttr if you can revert the changes here, would be great to merge, thanks!

**Review by zlangbert — CHANGES_REQUESTED** [review]

File: custom_components/daikinone/daikinone.py (hunk 1 above)
Comment (wirecuttr) [reply custom_components/daikinone/daikinone.py:292]: I'll do my best :). I'm pretty new to al of this. I merged two branches on my end, not realizing that it impacts the pull request!
